Library/Homebrew/vendor/bundle/ruby/2.6.0/gems/byebug-11.1.3/lib/byebug.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
```

Comment:
I would suggest gitignoring all of these as they aren't needed by non-developers.

Reply:
Done :)